internals/src/Tizen.Applications.ThemeManager/Tizen.Applications.ThemeManager/ThemeLoader.cs

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace Tizen.Applications.ThemeManager
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class ThemeLoader : IDisposable
+    {
+        private const string LogTag = "Tizen.Applications.ThemeManager";
+        private bool _disposed = false;
+        private event EventHandler<ThemeEventArgs> _changedEventHandler;
+        private Interop.ThemeManager.ThemeLoaderChangedCallback _callback;
+        private string _eventId;
+        private Theme _currentTheme = null;
+        internal IntPtr _loaderHandle = IntPtr.Zero;
+
+        /// <summary>
+        /// Creates ThemeLoader.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <exception cref="OutOfMemoryException">Failed to create handle.</exception>
+        public ThemeLoader()
+        {
+            Interop.ThemeManager.ErrorCode err = Interop.ThemeManager.LoaderCreate(out _loaderHandle);
+            if (err != Interop.ThemeManager.ErrorCode.None)
+            {
+                Interop.ThemeManager.ThemeManagerErrorFactory.GetException(err, "Failed to create themeloader");
+            }
+        }
+
+        private void OnEvent(IntPtr handle, IntPtr userData)
+        {
+            _changedEventHandler?.Invoke(null, new ThemeEventArgs(handle)
+            {
+            });
+        }
+
+        /// <summary>
+        /// Adds or removes events for theme changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of out of memory.</exception>
+        public event EventHandler<ThemeEventArgs> Event
```

Comment:
`Event` is not proper naming for event
```suggestion
        public event EventHandler<ThemeEventArgs> ThemeChanged
```

Reply:
Done